src/sentry/uptime/endpoints/project_uptime_alert_index.py

```diff
@@ -0,0 +1,63 @@
+from drf_spectacular.utils import extend_schema
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases import ProjectEndpoint
+from sentry.api.serializers import serialize
+from sentry.apidocs.constants import (
+    RESPONSE_BAD_REQUEST,
+    RESPONSE_FORBIDDEN,
+    RESPONSE_NOT_FOUND,
+    RESPONSE_UNAUTHORIZED,
+)
+from sentry.apidocs.parameters import GlobalParams
+from sentry.models.project import Project
+from sentry.uptime.endpoints.serializers import ProjectUptimeSubscriptionSerializer
+from sentry.uptime.endpoints.validators import UptimeMonitorValidator
+
+
+@region_silo_endpoint
+@extend_schema(tags=["Uptime Monitors"])
+class ProjectUptimeAlertIndexEndpoint(ProjectEndpoint):
+    publish_status = {
+        "POST": ApiPublishStatus.PUBLIC,
```

Comment:
I know the flag will bar customers from creating the monitors if they don't have it, but is it necessary to publish to our docs right now?

Reply:
Moved this over to experimental